sdcm/mgmt.py

```diff
@@ -345,6 +345,19 @@ def update(self, name=None, host=None, ssh_identity_file=None, ssh_user=None, cl
             cmd += " --ssl-user-cert-file {} --ssl-user-key-file {}".format(SSL_USER_CERT_FILE, SSL_USER_KEY_FILE)
         res = self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
 
+    def delete_task(self, task_id):
+        cmd = "-c {} task delete {}".format(self.id, task_id)
+        logger.debug("Task Delete command to execute is: {}".format(cmd))
+        stdout, stderr = self.sctool.run(cmd=cmd, parse_table_res=False)
+        if stderr:
+            logger.warning("stderr:\n" + stderr)
+            raise ScyllaManagerError("Unknown failure for sctool '{}' command".format(cmd))
+        logger.debug("Deleted the task '{}' successfully!". format(task_id))
+
+    def delete_automatic_repair(self):
+        repair_task = self.repair_task_list[0]
```

Comment:
How do we know that the first item in task list is the automatic repair, which is caused by adding cluster to manager?

Reply:
Can we verify it by checking the `next run` column?

Reply:
I agree with amos here, we shouldn't make assumptions that blow up later.
Let's run "task list", identify the correct task by something more deterministic and then delete it.

Reply:
You're both right, though verifying which is the automatic repair through `next run` is not correct either. So I decided instead to add an option to delete the repair in the add_cluster function. There, since the cluster was just added, there would be only one repair task, so it will be safe to assume it is the automatic one.
I added the change in the recently force-pushed commit, please review it @roydahan @amoskong